README.md

```diff
@@ -71,11 +71,11 @@ See [the list of releases][6] to find out about feature changes.
 
 [24]: https://groups.google.com/forum/#!forum/projectvelero
 [25]: https://kubernetes.slack.com/messages/velero
-[26]: https://github.com/heptio/velero/blob/master/docs/zenhub.md
+[26]: https://velero.io/docs/master/zenhub
 
 
-[29]: https://heptio.github.io/velero/
-[30]: /docs/troubleshooting.md
+[29]: https://velero.io/docs/master/
```

Comment:
@jonasrosland it would be nice if rather than directing to the "master" docs we had a stable link that would get the user to the latest released version - but doesn't look like `velero.io/docs/` will do it - is there any way to do this? that's how the previous site worked.